lib/credentials.js

```diff
@@ -70,6 +83,12 @@ Credentials.prototype._expand = function() {
     this.requestPubKey = requestDerivation.publicKey.toString();
   }
   var network = WalletUtils.getNetworkFromXPubKey(this.xPubKey);
+  if (typeof this.externalSource == "string") {
+    var xPrivKey = new Bitcore.HDPrivateKey(network);
```

Comment:
No need to sign it now. We will do it when we implement the changes above. Thanks!